templates/compose/rybbit.yaml

```
@@ -0,0 +1,138 @@
+# documentation: https://rybbit.io/docs
+# slogan: Open-source, privacy-first web analytics.
+# tags: analytics,web,privacy,self-hosted,clickhouse,postgres
+# logo: svgs/rybbit.svg
+# port: 3000
+
+services:
+  rybbit:
+    image: 'ghcr.io/rybbit-io/rybbit-client:latest'
+    container_name: rybbit-client
+    environment:
+      - NODE_ENV=production
+      - 'NEXT_PUBLIC_BACKEND_URL=${SERVICE_URL_RYBBIT}'
+      - 'NEXT_PUBLIC_DISABLE_SIGNUP=${DISABLE_SIGNUP:-false}'
+    depends_on:
+      - rybbit_backend
+    restart: unless-stopped
+
+  rybbit_backend:
+    image: 'ghcr.io/rybbit-io/rybbit-backend:latest'
+    container_name: rybbit-backend
+    environment:
+      - NODE_ENV=production
+      - TRUST_PROXY=true
+      - 'BASE_URL=${SERVICE_URL_RYBBIT}'
+      # --- Coolify-Managed Secrets ---
+      - 'CLICKHOUSE_PASSWORD=${SERVICE_PASSWORD_CLICKHOUSE}'
+      - 'POSTGRES_PASSWORD=${SERVICE_PASSWORD_POSTGRES}'
+      - 'BETTER_AUTH_SECRET=${SERVICE_BASE64_64_BACKEND}'
+      # --- User Settings ---
+      - 'DISABLE_SIGNUP=${DISABLE_SIGNUP:-false}'
+      - 'DISABLE_TELEMETRY=${DISABLE_TELEMETRY:-true}'
+      # --- Internal Config ---
+      - 'CLICKHOUSE_HOST=http://rybbit_clickhouse:8123'
+      - CLICKHOUSE_USER=default
+      - CLICKHOUSE_DB=analytics
```

Since the user and db is already hard coded on `rybbit_clickhouse` service we don't have to hard code it here (in future if we need to change these value we just need to change in one place)
```suggestion
      - CLICKHOUSE_USER=${CLICKHOUSE_USER}
      - CLICKHOUSE_DB=${CLICKHOUSE_DB}
```